lib/config/src/app/mod.rs

```diff
@@ -234,7 +234,16 @@ pub struct AppConfigCapabilityInstaBootV1 {
     ///
     /// NOTE: if no requests are configured, then a single HTTP
     /// request to '/' will be performed instead.
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
     pub requests: Vec<HttpRequest>,
+
+    /// Maximum age of snapshots.
+    ///
+    /// Format: 5m, 1h, 2d, ...
+    ///
+    /// Snapshots older than this value will be automatically purged.
```

Comment:
Note: from this I understood that instaboot will not work after max-age. We should rephrase this

Reply:
Re-worded the comment.